hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,18 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+    if s.communicator.rank == 0:
+        s.configuration.box = [1, 1, 1, 0, 0, 0]
+        # set up particles outside of the box
+        s.particles.N = 2
+        s.particles.position[:] = [[1.5, 0, 0], [0.3, 0.2, 0]]
+        # wrap
+        s.wrap()
+        # check the particles are where they should be
+        assert numpy.all(s.particles.position == [[0.5, 0, 0], [0.3, 0.2, 0]])
```

Comment:
I would test all periodic directions, just to be on the safe side.

Reply:
This turned out to be a good call :)
Will add those tests